esbuild.config.mjs

```diff
@@ -54,7 +55,7 @@ const esBuildOptions = {
     GlobalsPlugin(globals),
   ],
   external: [
-    'react',
+    '^react$', // Use regex ^$ because we do want to bundle 'react/jsx-transform'
```

Comment:
MUI 5 and emotion ship code that uses the automatic JSX transform, so their source code -> distribution looks kind of like this:
```jsx
	<Component {...props}>
		{children}
	</Component>
```

```js
	import jsx from 'react/jsx-transform'
	jsx(Component, props, children) // instead of React.createElement
```

We have `react` set as an external package, and the React UMD bundle doesn't include the JSX transform module, so this would break. Using regex allows us to keep `'react'` imports external but still bundle `'react/jsx-transform'`, fixing this case.